app/views/courses/show.html.erb

```diff
@@ -9,17 +9,17 @@
 <div data-module="ga-track-scrolling ga-track-copying">
   <h1 class="govuk-heading-xl">
     <span class="govuk-heading-l govuk-!-margin-bottom-0" data-qa="course__provider_name">
-      <%= smart_quotes(course.provider.provider_name) %>
+      <%= smart_quotes(course.provider.name) %>
     </span>
     <%= course.name_and_code %>
   </h1>
 
-  <p class="govuk-body-l" data-qa='course__description'><%= course.description %></p>
+  <p class="govuk-body-l" data-qa='course__description'><%= course.summary %></p>
```

Comment:
data-qa=`course__description`?

Reply:
👍

Reply:
Out of interest why is this  course__description and not course__summary (I'm reviewing per commit so this might become obvious)